dat/outfits/bioship/skills/silence.lua

```diff
@@ -0,0 +1,18 @@
+
+function onstealth( _p, po, stealthed )
+   if stealthed then
+      po:state( "on" )
+   else
+      po:state( "off" )
+   end
+end
+
+function update( p, po )
```

Comment:
Strictly speaking, it shouldn't need an update function, should it? Just catching the stealth event should be enough.